maxim/logger/mistral/client.py

```
@@ -39,6 +39,22 @@ def _setup_logging(
+                        continue
+                    if isinstance(content, str):
+                        input_message = content
+                        break
+                    if isinstance(content, list):
+                        for item in content:
+                            if isinstance(item, dict) and item.get("type") == "text":
+                                input_message = item.get("text", "")
+                                break
+            if input_message is not None:
+                trace.set_input(input_message)
         except Exception as e:
```

Ruff (0.11.9)

Do not catch blind exception: `Exception`

(BLE001)